src/App.js

```diff
@@ -176,7 +176,13 @@ function App() {
             <Route exact path="/project-updates/1">
               <ProjectUpdates />
             </Route>
-          </React.Fragment>
+            <Route exact path="/posts/:slug">
+              <Post />
+            </Route>
+            <Route>
+              <Post />
+            </Route>
```

Comment:
Is this here on purpose?

Reply:
Ok I think I understand. It's here to catch any route that isn't defined and render the 404. 
It's nice, but i'd prefer something more explicit. Like instead of <Post />, render a <404/> component which can simply wrap the FourOhFour component.